### ISSUE TITLE
Can you provide an example jumprope dataset file. ?

### ISSUE THREAD

**Issue by KhushalGupta** [csaden/lps-dashboard#4]
Can you provide an example jumprope dataset file. ?

**Comment by csaden** [comment]
No, I can't because the data set contains personal information on students. The schema section on the README explains what's needed to render data.

**Comment by KhushalGupta** [comment]
I tried to make a csv file, as per the schema section, but when i upload it in the application. the file is not readed correctly.

Can you please at the attachment in this comment, and provide me the correct format with some sample data. ?

Thank You.


[Data.zip](https://github.com/csaden/lps-dashboard/files/3331957/Data.zip)

**Comment by csaden** [comment]
Sorry for the delayed response. Here's some sample data, some fields have been anonymized. Let me know if you have other questions. Are you seeing specific errors in the console when you try to start up the dashboard?


AssessmentID | AssessmentTitle | AssessmentTitleRaw | CourseName | FirstName | LastName | Score | Section | StandardID | StandardTitle | StandardType | StudentID | TeacherName | Weight
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
`Numerical ID` | [Project] Elements & Principles 1 - 2017-09-05 | Elements & Principles 1 | AP Studio Art: Drawing (A) | `FirstName` |`LastName` | 88 | #28934 | 53832610 | Students will be able to demonstrate understanding of the elements of art and principles of design through an original composition. | Academic | `Numerical ID` | `Teacher Name` | 1
